course05/homework01/freekpaans/string.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <string>
+#include <ostream>
+#include <istream>
```

Comment:
These headers are quite heavy and you should not include them in your header file; especially if they are not used! You only need declarations of streams, not their definitions, so `iosfwd` is what you want (and these in the .cc file - another argument to not define stream ops in the header file!)